### ISSUE TITLE
[Stats] - Accessibility - When larger fonts are used, some text are missing or cut off

### ISSUE THREAD

**Issue by jostnes** [woocommerce/woocommerce-ios#8383]
**Describe the bug**
When using larger fonts, some of the text on the stats area is cut off or missing. Screenshots below.

**To Reproduce**
1. Change device font size to a larger font (try a few fonts and the largest)
2. Go to the charts on My store
3. Change to different tabs
4. See that some text (revenue, dates) are missing or cut off

**Screenshots**
<img src="https://user-images.githubusercontent.com/17252150/207000140-f029739c-3a34-4db1-9179-f4616427b0ab.PNG" width=50% height=50%>
<img src="https://user-images.githubusercontent.com/17252150/207000191-fd5a13cb-2a0d-423a-aa4b-870b7edc4960.PNG" width=50% height=50%>
<img src="https://user-images.githubusercontent.com/17252150/207000210-061a9c32-1a95-4b2b-974a-f7f69dd4779f.PNG" width=50% height=50%>
<img src="https://user-images.githubusercontent.com/17252150/207000223-73948dfd-c96b-437c-a6d7-b51de2fbd7fd.PNG" width=50% height=50%>

**Expected behavior**
The text should be displayed and readable. 

**Isolating the problem (mark completed items with an [x]):**
- [x] I have deactivated other plugins and confirmed this bug occurs when only WooCommerce plugin is active.
- [x] This bug happens with a default WordPress theme active, or [Storefront](https://woocommerce.com/storefront/).
- [x] I can reproduce this bug consistently using the steps above.

**Mobile Environment**
Please include:
- Device: iPhone 13 mini
- iOS version: 15.7.1
- WooCommerce iOS version: 11.4 (also tested in beta v11.6)

**WordPress Environment**
<details>
```
Copy and paste the system status report from **WooCommerce > System Status** in WordPress admin.
```
</details>
